array/view.mbt

```diff
@@ -27,3 +27,57 @@ pub fn each[T](self : ArrayView[T], f : (T) -> Unit) -> Unit {
     f(self[i])
   }
 }
+
+///|
+pub fn eachi[T](self : ArrayView[T], f : (Int, T) -> Unit) -> Unit {
+  for i, v in self {
+    f(i, v)
+  }
+}
+
+///|
+pub fn all[T](self : ArrayView[T], f : (T) -> Bool) -> Bool {
+  for v in self {
+    if f(v).not() {
```

Comment:
Judging from: https://github.com/moonbitlang/core/pull/1324
this should probably be:

```suggestion
    if not(f(v)) {
```

Reply:
Thanks for your reminder. Solved.